frontend/package.json

```diff
@@ -187,6 +187,7 @@
     "react-transition-group": "2.3.x",
     "react-virtualized": "9.x",
     "redux": "4.0.1",
+    "redux-thunk": "2.4.0",
```

Comment:
@andrewballantyne curious on your thoughts long term here (should not be addressed in this PR): should we make react/redux peer dependencies over dependencies? Since only one version of react can ever be on the page, I'd imagine defining react as a dependency in the SDK could cause issues down the road where multiple versions of React end up on a page.

Reply:
@invincibleJai do we know which webpack chunk this is pulled into? I'm curious if there is a difference in bundle size.

Reply:
there is no difference in bundle size and here I just replaced the copied code from redux-thunk to actual dependency.

https://github.com/openshift/console/blob/master/frontend/public/redux.ts#L12-L28

I need to check which chunk it goes in but should be the same as we have for `react` or `patternfly` , as added in shared modules for SDK as well 

https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/src/shared-modules.ts#L4-L14

Reply:
@florkbr I think the App owns the state, @vojtechszocs how do we handle App dependencies vs SDK dependencies vs Plugin dependencies?

I believe this is already been discussed but I don't recall the details of the conversation.